requirements_qa.txt

```diff
@@ -3,4 +3,4 @@ matplotlib==3.3.4
 numpy==1.20.1
 pylint==2.6.0
 pyspelling==2.7.3
-pytest==6.2.5
+pytest==6.2.5
```

Comment:
discard any changes here

Reply:
It was accidental, I returned the original.